### ISSUE TITLE
plans to use the arrow keys?

### ISSUE THREAD

**Issue by houblon** [dcooney/a11ydrops#1]
I notice you have the key codes for the arrows in keycodes.js, but they aren't used in methods.js. Are you planning to implement these keys, or are you planning to leave that to devs to customize?

**Comment by dcooney** [comment]
@houblon Yes, I plan to add arrowkey controls at some point in the future.
